content/faqs.yaml

```diff
@@ -1,13 +1,32 @@
 - header: Frequently Asked Questions
   faqs:
-    - header: Who can become a member?
-      body: The clubs is open to any student, ranging from novice developers who are just starting, to advanced developers who want to further their skills.
+    - header: What do we do?
+      body: We hold social tech meetups, tech talks, workshops and hackathons!
 
-    - header: How frequently do events and workshops occur?
-      body: We hold workshops and showcases at the University of Auckland & we would recommend you to sign up using our Google form and liking our Facebook page to get updates.
+    - header: Can I join if I have no experience?
+      body: |
+        Absolutely, we welcome everyone from any level of experience and from any discipline!
+        Most of our talks and workshops are from scratch so lack of experience is definitely not a problem.
 
-    - header: What should I carry when attending a workshop?
-      body: We recommend you carry with you a notebook, pen and a laptop because more often than not we make our hands dirty with code. Most importantly, carry along a healthy dose of curiosity and enthusiasm.
+    - header: Is it free to sign up?
+      body: |
+        All our events are 100% free and it's free to join!
+        BTW they all have FREE PIZZA 🍕!!
+
+    - header: How is it different to other tech clubs?
+      body: |
+        We aim to hold a lot more tech-focused social events like our tech meetups which is purely a social event
+        to bring people interested in tech together.
```

Comment:
We also get speakers from Google (e.g. we provide more tech-talk style events, compared to DEVS which does workshops101 and WDCC which is projects based)